components/support/images/src/main/res/values/dimens.xml

```diff
@@ -2,6 +2,7 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <dimen name="mozac_support_images_size_default">102dp</dimen>
+    <dimen name="mozac_support_images_size_max" tools:ignore="PxUsage">99999px</dimen>
```

Comment:
Interchanging px values with dp seems wrong. Following along the code, these values are used in the [`AndroidImageDecoder`][0] for the [`inSampleSize`][1] Option used with the `BitmapFactory.decodeByteArray` call, so that seems to expect a px value but for the default cause we can't provide that since it depends on the device.

`getDimensionPixelSize` gives us the px value from dp but looking at the docs, it's unclear whether it can handle being provided a px value accurately. Why not just provide a dp value that equates to a max px like 99999dp?

[0]: https://github.com/mozilla-mobile/android-components/blob/cdef87e847f4272fcb43e441794013e37feb97b9/components/support/images/src/main/java/mozilla/components/support/images/decoder/AndroidImageDecoder.kt#L85
[1]: https://developer.android.com/reference/android/graphics/BitmapFactory.Options#inSampleSize